### ISSUE TITLE
Add method for downloading orders

### ISSUE THREAD

**Issue by theted** [Raviteq/adrapid-api#5]
Since HTML5-banners consists of multiple files, there needs to be a way of downloading a set of files - preferably as a .`zip` file. Another use-case is for downloading a complete order containing multiple items/formats.

I would suggest a `download order` method, which returns the path to the zip files, or alternatively sets up headers for a _force download_ of the zip file (for client-side/browser implementation).

**Suggested usage**
`api/download/:order` - Download .zip for whole order where `order` is the id of the order.

`api/download/:order/:item` - Download only single order item.

**Comment by manast** [comment]
I dont think we need an API for this. Since an order has already the urls to each item and the complete order. By doing this we do not need to commit to a given URL for a downloading.

**Comment by theted** [comment]
I am thinking primarily of the HTML5-banners, since those ads are compromised af multiple files (the html file, javascripts, images).

If a third party for example would like to download and restribute a HTML5-banner elsewhere, they would have to figure out the location of the individual files themselves, since we only provide the URL of the html index file. A download method would do this for them, clients would also be able to downlad a complete order or a item only.

Since we already have implemented a function for generating .zip files for whole orders, this would be quite easy to implement. I agree that this should not be a priority, putting this issue in the `wishlist` milestone for now.

**Comment by manast** [comment]
I think you misunderstood me. I agree that HTML5 results should be delivered as .zip files, although we can have a URL for a preview without .zip, just so show the result. But we dont need special API for retrieving the .zips they should be accessible in the GET order api, in the list of URLS for the results...
